AttributeError: 'list' object has no attribute 'asList'

**Describe the bug**
Komodo bleeding fails with many of the following error messages:
```
/tmp/jenkins-komodo-f_scout_ci/workspace/unit-tests/semeio/test-root/tests/jobs/misfit_preprocessor/test_integration.py:242: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/prog/res/komodo/bleeding-py36-rhel7/root/lib/python3.6/site-packages/semeio/communication/semeio_script.py:72: in run_with_handler
    self._real_run(*args, **kwargs)
/prog/res/komodo/bleeding-py36-rhel7/root/lib/python3.6/site-packages/semeio/workflows/misfit_preprocessor/misfit_preprocessor.py:43: in run
    CorrelatedObservationsScalingJob(self.ert()).run(scaling_configs)
/prog/res/komodo/bleeding-py36-rhel7/root/lib/python3.6/site-packages/semeio/communication/semeio_script.py:72: in run_with_handler
    self._real_run(*args, **kwargs)
/prog/res/komodo/bleeding-py36-rhel7/root/lib/python3.6/site-packages/semeio/workflows/correlated_observations_scaling/cos.py:172: in run
    scale_observations(obs, scale_factor, config.get_update_keys())
/prog/res/komodo/bleeding-py36-rhel7/root/lib/python3.6/site-packages/semeio/workflows/correlated_observations_scaling/update_scaling.py:16: in scale_observations
    update_data = create_active_lists(obs, obs_list)
/prog/res/komodo/bleeding-py36-rhel7/root/lib/python3.6/site-packages/semeio/workflows/correlated_observations_scaling/obs_utils.py:69: in create_active_lists
    obs_index = _data_index_to_obs_index(enkf_observations, event.key, event.index)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

obs = EnkfObs(valid, len=8) at 0x578f680, obs_key = 'WPR_DIFF_1'
data_index_list = (400, 800, 1200, 1800)

    def _data_index_to_obs_index(obs, obs_key, data_index_list):
        if obs[obs_key].getImplementationType().name != "GEN_OBS":
            return data_index_list
        elif data_index_list is None:
            return data_index_list
    
>       for timestep in obs[obs_key].getStepList().asList():
E       AttributeError: 'list' object has no attribute 'asList'

/prog/res/komodo/bleeding-py36-rhel7/root/lib/python3.6/site-packages/semeio/workflows/correlated_observations_scaling/obs_utils.py:132: AttributeError1
```

**To Reproduce**
TBD


**Expected behavior**
Not fail

**Additional context**
Komodo bleeding, full report: https://ci.equinor.com/komodo/job/unit-tests/job/semeio/1000/console

--- a/semeio/workflows/ahm_analysis/ahmanalysis.py
+++ b/semeio/workflows/ahm_analysis/ahmanalysis.py
@@ -12,7 +12,7 @@ from scipy.stats.stats import ks_2samp
 from sklearn.decomposition import PCA
 from sklearn.preprocessing import StandardScaler
 
-from xtgeo.grid3d import Grid
+import xtgeo
 from xtgeo.grid3d import GridProperty
 from semeio.communication import SemeioScript
 from semeio._exceptions.exceptions import ValidationError
@@ -340,7 +340,9 @@ def _get_field_params(ert, ensemble_size, field_parameters, target_name):
 
 
 def _import_field_param(input_grid, param_name, files):
-    grid_param = Grid(input_grid.rsplit(".", 1)[0], fformat="eclipserun")
+    grid_param = xtgeo.grid_from_file(
+        input_grid.rsplit(".", 1)[0], fformat="eclipserun"
+    )
     all_input = []
     for file_path in files:
         proproff = GridProperty(file_path, name=param_name, grid=grid_param)
@@ -455,7 +457,7 @@ def load_grid_to_dataframe(grid_path):
     """Get field grid characteristics/coordinates"""
     grid_path = Path(grid_path).with_suffix("")
     try:
-        grid = Grid(grid_path, fformat="eclipserun")
+        grid = xtgeo.grid_from_file(grid_path, fformat="eclipserun")
         return grid.dataframe(activeonly=False)
     except OSError as err:
         raise OSError("A grid with .EGRID format is expected.") from err

--- a/semeio/workflows/correlated_observations_scaling/obs_utils.py
+++ b/semeio/workflows/correlated_observations_scaling/obs_utils.py
@@ -129,7 +129,7 @@ def _data_index_to_obs_index(obs, obs_key, data_index_list):
     elif data_index_list is None:
         return data_index_list
 
-    for timestep in obs[obs_key].getStepList().asList():
+    for timestep in obs[obs_key].getStepList():
         node = obs[obs_key].getNode(timestep)
         index_map = {node.getIndex(nr): nr for nr in range(len(node))}
     return [index_map[index] for index in data_index_list]

--- a/semeio/workflows/correlated_observations_scaling/update_scaling.py
+++ b/semeio/workflows/correlated_observations_scaling/update_scaling.py
@@ -25,7 +25,7 @@ def _update_scaling(obs, scale_factor, obs_list):
     """
     for event in obs_list:
         obs_vector = obs[event.key]
-        step_list = list(obs_vector.getStepList())  # List of steps, 1-indexed
+        step_list = obs_vector.getStepList()  # List of steps, 1-indexed
         for step in step_list:
             obs_node = obs_vector.getNode(step)
             if obs_vector.getImplementationType().name == "SUMMARY_OBS":

--- a/setup.py
+++ b/setup.py
@@ -49,7 +49,7 @@ setup(
     ],
     install_requires=[
         "ecl",
-        "ert >= 2.27.0rc1",
+        "ert >= 2.31.0b0",
         "configsuite>=0.6",
         "numpy",
         "pandas>0.24",

--- a/tests/jobs/localisation/test_integration.py
+++ b/tests/jobs/localisation/test_integration.py
@@ -292,8 +292,7 @@ def create_box_grid_with_inactive_and_active_cells(
     zinc = 10.0
     xori = 0.0
     yori = 0.0
-    grid = xtgeo.Grid()
-    grid.create_box(
+    grid = xtgeo.create_box_grid(
         dimension=(nx, ny, nz),
         origin=(xori, yori, 0.0),
         increment=(xinc, yinc, zinc),

